tests/test_funs.py

```diff
@@ -35,13 +37,14 @@ def test_solutions_with_orthogonal_map(self):
         err_map = sol_map - t
 
         # Assert
-        assert np.linalg.norm(err_mud) < 1e-8
-        assert np.linalg.norm(err_alt) < 1e-8
+        assert np.linalg.norm(err_mud) < 1e-6
+        assert np.linalg.norm(err_alt) < 1e-6
         assert np.linalg.norm(err_mud) < np.linalg.norm(err_map)
 
     def test_updated_cov_has_R_equal_zero_for_full_rank_A(self):
         up_cov = mdf.updated_cov(self.A, self.id, self.id)
-        assert np.linalg.norm(up_cov - np.linalg.inv(self.A.T @ self.A)) < 1e-6
+        absolute_error = np.linalg.norm(up_cov - np.linalg.inv(self.A.T @ self.A))
+        assert absolute_error / len(up_cov) < 1e-12
```

Comment:
@eecsu I have these tests kick off weekly and this one just failed. I was curious how long it would take. The systems CI runs on are can be less precise than my home dev server, but in this case it was a matter of "correct order of magnitude" (3E-12) so the processor is probably capable of the same and I should lower this bound to something like `(1E-10)` to be safer.

Reply:
could settle on 1E-8 as a middle ground for before / after this change.

Reply:
re-running I think I got allocated an older processor because the error went to 1E-9, so I think I need to go back to 1E-8